cpp/src/arrow/compute/api_vector.h

```diff
@@ -188,6 +188,35 @@ class ARROW_EXPORT PartitionNthOptions : public FunctionOptions {
   NullPlacement null_placement;
 };
 
+/// \brief Options for IsMonotonic
+class ARROW_EXPORT IsMonotonicOptions : public FunctionOptions {
+ public:
+  enum NullHandling {
+    /// Ignore nulls.
+    IGNORE_NULLS,
```

Comment:
Based on the other enum names, use only `IGNORE`, since `enum NullHandling` already specifies this is for nulls.

Reply:
I had that initially, but IGNORE caused compilation issues on Windows.

Reply:
Well, nevermind.